test/verifiedTests/visualization/testEFMviz/testEfmEnrichmentAnalysis.m

```diff
@@ -0,0 +1,46 @@
+% The COBRAToolbox: <testNameOfSrcFile>.m
```

Comment:
Same as the other test files, please add in the missing information in the header